packages/sirius-web/backend/sirius-web-application/src/main/resources/schema/explorer.graphqls

```diff
@@ -8,4 +8,13 @@ input ExplorerEventInput {
   editingContextId: ID!
   expanded: [String!]!
   activeFilterIds: [String!]!
+}
+
+extend type EditingContext {
```

Comment:
on TreeDescription

Reply:
Discussed that this query should remain on EditingContext